database/generateData.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable func-names */
+const faker = require('faker');
+const db = require('./index');
+
+// generate fake data for 100 unique businesses
+const generateBusiness = function () {
```

Comment:
You can consider having `generateBusiness` and other data generation functions take in an optional argument that determines how much data you want to generate.
```suggestion
const generateBusiness = function (amount = 100) {
```

Reply:
Right! Default parameters. Made the change.